app/helpers/import_file_helper.rb

```diff
@@ -18,6 +18,9 @@ def self.create_new_user(attributes, session)
     user.parent_id = (session[:user]).id
     user.timezonepref = User.find(user.parent_id).timezonepref
     user.save!
+    password = 'qwerty' # the password is reset
```

Comment:
I remember the password is auto-generated instead of hard coded after importing new students.